### ISSUE TITLE
Case of replicate column names differ among years

### ISSUE THREAD

**Issue by rluedde** [cenpy-devs/cenpy#116]
``` Python
>>> from cenpy.moe import replicate_table_utils as crtu
>>> data_14 = crtu.get_replicate_data_api(["B15002"], 2014, "140", "04")
https://www2.census.gov/programs-surveys/acs/replicate_estimates/2014/data/5-year/140/B15002_04.csv.gz
>>> data_18.columns.levels[0][:3]
Index(['ESTIMATE', 'MOE', 'SE'], dtype='object', name='categories')
>>> data_14.columns.levels[0][:3]
Index(['estimate', 'moe', 'SE'], dtype='object', name='categories')
```
In the 2014 data, the names of these columns are capitalized and all other years (I think) are lowercase. This causes issues in (at least) [apply_func](https://github.com/cenpy-devs/cenpy/blob/fde2ad6c71d4a81aab6d8950d76c39b36b9d377c/cenpy/moe/replicate_table_utils.py#L452).

I don't have good ideas for a solution. I imagine it's a `.upper()` or `.lower()` call somewhere in [read_replicate_file](https://github.com/cenpy-devs/cenpy/blob/fde2ad6c71d4a81aab6d8950d76c39b36b9d377c/cenpy/moe/replicate_table_utils.py#L42) or [get_replicate_data](https://github.com/cenpy-devs/cenpy/blob/fde2ad6c71d4a81aab6d8950d76c39b36b9d377c/cenpy/moe/replicate_table_utils.py#L102). A comprehension might be needed as well?

**Comment by dfolch** [comment]
Good idea to always convert to the same case. It is probably not worth the extra code to _test_ for capitalization, just run `upper` on these three column names for all files read in with `read_replicate_file`.
